internal/oem/oem.go

```diff
@@ -76,6 +77,7 @@ func (c Config) DefaultUserConfig() types.Config {
 }
 
 var configs = registry.Create("oem configs")
+var yes = util.BoolToPtr(true)
```

Comment:
Yeah, it seemed cleaner to not have it open-coded all over the place.  Ready to merge from my end.